lms/djangoapps/grades/tests/test_new.py

```diff
@@ -239,56 +240,167 @@ def test_save_and_load(self):
 
 
 @ddt.ddt
-class TestMultipleProblemTypesSubsectionScores(ModuleStoreTestCase, ProblemSubmissionTestMixin):
+class TestMultipleProblemTypesSubsectionScores(SharedModuleStoreTestCase):
     """
     Test grading of different problem types.
     """
 
-    default_problem_metadata = {
-        u'graded': True,
-        u'weight': 2.5,
-        u'max_score': 7.0,
-        u'due': datetime.datetime(2099, 3, 15, 12, 30, 0, tzinfo=pytz.utc),
-    }
-
-    COURSE_NAME = u'Problem Type Test Course'
-    COURSE_NUM = u'probtype'
+    SCORED_BLOCK_COUNT = 6
```

Comment:
Yup.  These are just for clarifying some mysterious numbers in the test results.  I tried to document them well, so if somebody alters the sample data, they know what to do here.